src/frontend/pages/Products/components/Filters.jsx

```diff
@@ -0,0 +1,224 @@
+        value={productState.price}
+        onChange={(e) =>
+          productDispatch({ type: "PRICE", payload: e.target.value })
+        }
+      />
+      <p>0-{productState.price}</p>
+      <h3 className="pt-1 pb-1">Category</h3>
+
+      <div>
+        <input
+          type="checkbox"
+          name="laptops"
+          id="laptops"
+          onChange={() =>
+            productDispatch({ type: "CATEGORY", payload: "Laptops" })
+          }
+          checked={productState.categories.includes("Laptops") ? true : false}
+        />
+        <label htmlFor="laptops">Laptops</label>
+      </div>
+      <div>
+        <input
+          type="checkbox"
+          name="smartphones"
+          id="smartphones"
+          onChange={() =>
+            productDispatch({ type: "CATEGORY", payload: "Smart phones" })
+          }
+          checked={
+            productState.categories.includes("Smart phones") ? true : false
+          }
+        />
+        <label htmlFor="smartphones">Smart phones</label>
+      </div>
+      <div>
+        <input
+          type="checkbox"
+          name="dslr"
+          id="dslr"
+          onChange={() =>
+            productDispatch({ type: "CATEGORY", payload: "DSLR" })
+          }
+          checked={productState.categories.includes("DSLR") ? true : false}
+        />
+        <label htmlFor="dslr">DSLR</label>
+      </div>
+      <div>
+        <input
+          type="checkbox"
+          name="keyboards"
+          id="keyboards"
+          onChange={() =>
+            productDispatch({ type: "CATEGORY", payload: "Keyboards" })
+          }
+          checked={productState.categories.includes("Keyboards") ? true : false}
+        />
+        <label htmlFor="keyboards">Keyboards</label>
+      </div>
+      <div>
+        <input
+          type="checkbox"
+          name="mouse"
+          id="mouse"
+          onChange={() =>
+            productDispatch({ type: "CATEGORY", payload: "Mouse" })
+          }
+          checked={productState.categories.includes("Mouse") ? true : false}
+        />
+        <label htmlFor="mouse">Mouse</label>
+      </div>
+      <div>
+        <input
+          type="checkbox"
+          name="headphones"
+          id="headphones"
+          onChange={() =>
+            productDispatch({ type: "CATEGORY", payload: "Headphones" })
+          }
+          checked={
+            productState.categories.includes("Headphones") ? true : false
+          }
+        />
+        <label htmlFor="headphones">Headphones</label>
+      </div>
+
+      <h3 className="pt-1 pb-1">Rating</h3>
+
+      <div>
+        <input
+          type="radio"
+          name="rating"
+          id="above4"
+          className="rating"
+          onChange={() => productDispatch({ type: "RATING", payload: 4 })}
+          checked={
+            productState.rating === ""
+              ? false
+              : productState.rating === 4
+              ? true
+              : false
```

Comment:
you can directly write checked= {productState.rating === 4}

Reply:
yeah thanks @romabulani, will change it